### PR TITLE
K8SPG-80 Add pg.percona.com group

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

File: build/pgo-apiserver/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/percona/percona-postgresql-operator
 
 COPY . .
 
-RUN GOBIN="$(pwd -P)/hack/tools" ./hack/update-codegen.sh
+#RUN GOBIN="$(pwd -P)/hack/tools" ./hack/update-codegen.sh
```
Comment (cap1984) [inline build/pgo-apiserver/Dockerfile:6]: pls)

Comment (nonemax) [reply build/pgo-apiserver/Dockerfile]: yeah, forgot to remove completely https://github.com/percona/percona-postgresql-operator/pull/84/commits/644ebe394c00be8d5a4215ae15a29c1a6fe00027

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/644ebe394c00be8d5a4215ae15a29c1a6fe00027
image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-644ebe39-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/57ecc6ef3464b392ffa2ed6344feff424e9e3d82
image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/57ecc6ef3464b392ffa2ed6344feff424e9e3d82
image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-57ecc6ef-pgo-deployer`

**Review by hors — CHANGES_REQUESTED** [review]
It does not work. I can't even start a new cluster.

**Comment by cap1984** [comment]
> It does not work. I can't even start a new cluster.

looks like Google has fixed their GKE engine

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| demand-backup | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/8f0f6a7548940aaeb20834108a232e71ceb05472
image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-8f0f6a75-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| demand-backup | failed |
| data-migration-gcs | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/50849c2f3ef49b7eef653877e5fb675f06dc59ba
image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-50849c2f-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| init-deploy | failed |
| data-migration-gcs | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/3857d6f5c4024eebb6489cbea4c65637390c2808
image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-3857d6f5-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| demand-backup | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/84e497ce14541a360135b2ccb3ca470e2a0752a0
image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-84e497ce-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |
| clone-cluster | failed |
| data-migration-gcs | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/84/commits/b5df8a9b709523b8b004253bcf778ec788dca0b6
image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-84-b5df8a9b-pgo-deployer`

**Review by hors — APPROVED** [review]

**Pull request merged**